src/pages-conditional/dapps.js

```diff
@@ -1134,7 +1134,7 @@ const DappsPage = ({ data, location }) => {
           <Translation id="get-started" />
         </H2>
         <p>
-          <Translation id="page-dapps-get-started-subtitle" />{" "}
+          <Translation id="page-dapps-get-started-subtitle" />
```

Comment:
I think this and the other changes got added accidentally from a different PR? 

Could you remove them from this PR please 🙏

Reply:
Hey @minimalsm , I have changed`getDefaultMessage` to `translateMessageId`.
>  Could you remove them from this PR please 🙏

Can you tell how to delete these changes from a pull request.
Thanks